config/opal_mca.m4

```diff
@@ -934,7 +936,7 @@ AC_DEFUN([MCA_COMPONENT_BUILD_CHECK],[
 
     # if we were explicitly disabled, don't build :)
     AS_IF([test "$DISABLE_$2" = "1"], [want_component=0])
-    AS_VAR_IF([DISABLE_$2_$3], [1], [want_component = 0])
+    AS_VAR_IF([DISABLE_$2_$3], ["1"], [want_component=0])
```

Comment:
I don't think this needs to be in quotes, however it works for me.

Reply:
ah yes you're correct.  not needed so removed (although left the white space cleanup)